src/ssl.c

```diff
@@ -13671,6 +13736,10 @@ int wolfSSL_GetSessionFromCache(WOLFSSL* ssl, WOLFSSL_SESSION* output)
             output->ticket = output->staticTicket;
             output->ticketLenAlloc = 0;
         }
+#ifdef WOLFSSL_TLS13
```

Comment:
`#if defined(HAVE_SESSION_TICKET) && defined(WOLFSSL_TLS13)`

Reply:
we are already in a region of code gated by HAVE_SESSION_TICKET